app/Repositories/PostRepository.php

```diff
@@ -120,7 +120,11 @@ public function create(array $data, $signupId)
     public function update($signup, $data)
     {
         if (array_key_exists('updated_at', $data)) {
-            $signup->fill(array_only($data, ['quantity', 'quantity_pending', 'why_participated', 'updated_at']));
+            if ($data['why_participated']) {
```

Comment:
Should we be checking that all of these values are filled in before overriding them in the database (not only `$data['why_participated']`)?

Reply:
We definitely need to allow `quantity` to be overwritten for quantity updates! 📈  `updated_at` might be leftover from the migration, I don't think anything else is telling us when it was updated at anymore?

Reply:
@katiecrane sorry I don't think I was clear! I mean check to see if `quantity` is filled in so we don't override this too? Or maybe this isn't a concern if `quantity` is always passed?

Reply:
@chloealee oh oops I was thinking the exactly the wrong thing. I'm not sure exactly how gambit does it, but I feel like I always get asked how many more nouns I've verbed?

Reply:
@aaronschachter can you confirm? The only thing that isn't asked again when a user reports back for the 2+ time with another post via SMS is the `why_participated`, is that correct?

Reply:
i think they said they only asked `why_participated` the first time, and subsequent posts just ask for an image and caption. I think adding a safety check to the `quantity` field to make sure it is passed before using a null value makes good sense.